tools/Azure.Mcp.Tools.Monitor/src/Commands/WebTests/WebTestsUpdateCommand.cs

```diff
@@ -0,0 +1,181 @@
+{
+    private const string CommandTitle = "Update a web test in Azure Monitor";
+
+    private readonly ILogger<WebTestsUpdateCommand> _logger = logger;
+
+    public override string Name => "update";
+
+    public override string Description =>
+        """
+        Updates an existing standard web test in Azure Monitor. Ping/Multstep web tests are deprecated and are not supported.
+        Returns the updated web test details.
+        """;
+
+    public override string Title => CommandTitle;
+
+    public override ToolMetadata Metadata => new()
+    {
+        Destructive = true,
+        Idempotent = true,
+        OpenWorld = false,
+        ReadOnly = false,
+        LocalRequired = false,
+        Secret = false
+    };
+
+    protected override void RegisterOptions(Command command)
+    {
+        base.RegisterOptions(command);
+
+        // Add required options
+        command.Options.Add(MonitorOptionDefinitions.WebTest.WebTestResourceName);
+        command.Options.Add(OptionDefinitions.Common.ResourceGroup.AsRequired());
+
+        // Add optional options - for update, most options become optional
+        command.Options.Add(MonitorOptionDefinitions.WebTest.AppInsightsComponentId.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.ResourceLocation.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.Locations.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.RequestUrl.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.WebTestName.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.Description.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.Enabled.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.ExpectedStatusCode.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.FollowRedirects.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.FrequencyInSeconds.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.Headers.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.HttpVerb.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.IgnoreStatusCode.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.ParseRequests.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.RequestBody.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.RetryEnabled.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.SslCheck.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.SslLifetimeCheckInDays.AsOptional());
+        command.Options.Add(MonitorOptionDefinitions.WebTest.TimeoutInSeconds.AsOptional());
+
+        command.Validators.Add(commandResult =>
```

Comment:
Is this the same validation you have in other commands?  If so, please reuse it.